src/lib/components/ActionTable.svelte

```diff
@@ -43,8 +44,11 @@
 			</tr>
 		</thead>
 		<tbody class="hover:[&>tr]:preset-tonal-primary">
-			{#each actions as action}
-				<tr>
+			{#each actions as action, index}
+				<tr
+					class={rowIndexHighlights[index] ? '[&&&]:preset-tonal-secondary' : ''}
```

Comment:
I don't think this priority hack is needed anymore

Reply:
Just tested it again to be absolutely sure and unfortunately the triple ampersand is the minimum number needed to bump the specificity enough to override the hover:[&>tr] class on the tbody.

However, here is another solution:

```
class={rowIndexHighlights[index] ? 'preset-tonal-secondary' : 'hover:preset-tonal-primary'}
```

This would take the hover: class off of the tbody and move it down to the tr, meaning every tr has the hover: class on it directly.

Up to you which you would prefer.